templates/terraform/examples/ml_job_training.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_ml_engine_job" "test_job" {
+  job_id          = "ml_job_training"
+  training_input  {
+    job_dir         = "gs://cio-strat-svc-acct-mgmt-238520-mlengine/scikit_learn_job_dir"
```

Comment:
When I run the test, I get this error:
```
Error 400: Field: job_dir Error: The provided GCS path gs://cio-strat-svc-acct-mgmt-238520-mlengine/scikit_learn_job_dir cannot be written by current user. Please make sure that the bucket exists and you have write access to it.
```

Is it possible to use other TF resources alongside this one to make sure the tests are able to run no matter the environment?

Reply:
yes, I will convert this to parameter

Reply:
Please pass correct data